src/Symfony/Component/Serializer/Tests/Normalizer/AbstractNormalizerTest.php

```diff
@@ -78,6 +78,9 @@ public function testGetAllowedAttributesAsString()
 
         $result = $this->normalizer->getAllowedAttributes('c', [AbstractNormalizer::GROUPS => ['*']], true);
         $this->assertEquals(['a1', 'a2', 'a3', 'a4'], $result);
+
+        $result = $this->normalizer->getAllowedAttributes('c', [AbstractNormalizer::GROUPS => ['_default']], true);
```

Comment:
Call the new constant here

Reply:
I don't like relying on constant directly concerning the tested class, in the test itself. Using the constant in the tests doesn't permit to detected BC Break in case someone change "_default", because the test will be change automatically :/

Reply:
it seems be alright to keep more isolate as possible, so writing the '_default' directly seems good IMO

Reply:
Should I mark this conversation as resolve ? And the same one below

Reply:
Yes